server/src/test/java/org/elasticsearch/indices/settings/UpdateNumberOfReplicasIT.java

```diff
@@ -159,6 +160,22 @@ public void testAutoExpandNumberOfReplicas0ToData() throws IOException {
         assertThat(clusterHealth.getIndices().get("test").getNumberOfReplicas(), equalTo(1));
         assertThat(clusterHealth.getIndices().get("test").getActiveShards(), equalTo(numShards.numPrimaries * 2));
 
+        if (randomBoolean()) {
+            client().admin().indices().prepareClose("test").setWaitForActiveShards(ActiveShardCount.ALL).get();
```

Comment:
Can we ack the close (or check that the index is effectively in close state after the following cluster health checks)?